docsrc/Managing_resources.rst

```diff
@@ -23,8 +23,10 @@ To get started, follow the steps below:
 
 	::
 
-		from firebolt.service.manager import ResourceManager
+		from firebolt.client import DEFAULT_API_URL
+		from firebolt.client.auth import ClientCredentials
 		from firebolt.common import Settings
+		from firebolt.service.manager import ResourceManager
 
 
 **2. Initialize a Settings object**
```

Comment:
Since we're deprecating Settings, let tell users to pass parameters directly in resource manager `__init__`.